bundle/compliance/cis_gcp/rules/cis_1_9/test.rego

```diff
@@ -0,0 +1,36 @@
+package compliance.cis_gcp.rules.cis_1_9
+
+import data.cis_gcp.test_data
+import data.compliance.policy.gcp.data_adapter
+import data.lib.test
+
+test_violation {
+	eval_fail with input as rule_input(["allUsers", "test.user@google.com"], "", "", {"state": "ENABLED"})
+	eval_fail with input as rule_input(["allAuthenticatedUsers"], "", "", {"state": "ENABLED"})
+	eval_fail with input as rule_input(["allUsers", "allAuthenticatedUsers"], "", "", {"state": "ENABLED"})
+}
+
+test_pass {
+	eval_pass with input as {"subType": "gcp-kms", "resource": {}}
```

Comment:
what is this resource?

Reply:
a resource without an `iam_policy`
Added a comment.